### PR TITLE
Refactor comments

### PR DESCRIPTION
**Pull request by max-yushchuk**

### CODE REVIEW

File: client/shared/comment/comment.component.js [hunk 1]
```diff
@@ -11,45 +11,58 @@ angular.module('comment', [])
         var MAX_COMMENT_CONTENT_LENGTH = 1000;
```
Comment (yurykozhenov) [inline client/shared/comment/comment.component.js]: Remove `MAX_COMMENT_CONTENT_LENGTH`

**Comment by yurykozhenov** [comment]
How can I test your component inside this branch?)

**Comment by max-yushchuk** [comment]
Wait Lena's place-detail-page or comment all content in place-detail.component and template. Include in place-detail.component module 'comment' and in that template:
`<comment input-object="$ctrl.place" input-object-type="'places'"></comment>`

**Comment by yurykozhenov** [comment]
**...** icon is too close to text

**Comment by yurykozhenov** [comment]
Restrict `textarea` so it can't be infinitely dragged down.
Maybe completely remove dragging possibility and make `textarea` adapt to how much text it contains

**Comment by yurykozhenov** [comment]
I think this buttons are too close to text box, they shouldn't stick to it.
Maybe add some margin?

**Comment by yurykozhenov** [comment]
Also, maybe make _disabled_ POST button darker? To give user some feedback.

File: client/assets/less/comment.less [hunk 2]
```diff
@@ -47,6 +47,9 @@
     }
     .show-comments-edit-content {
       margin-right: 15px;
+      div {
+        float: right;
```
Comment (yurykozhenov) [inline client/assets/less/comment.less]: You can replace this code with Bootstraps `pull-right` class on your `div`

Comment (max-yushchuk) [inline client/assets/less/comment.less]: Good Idea

**Pull request merged**